rs/nervous_system/candid_utils/src/printing.rs

```diff
@@ -0,0 +1,10 @@
+use candid::{CandidType, IDLValue};
+
+pub fn pretty<T: CandidType>(value: T) -> Result<String, String> {
+    let value = IDLValue::try_from_candid_type(&value).map_err(|err| err.to_string())?;
+
+    Ok(value.to_string())
```

Comment:
No change required.

This incantation is fairly small -> maybe, it is best to just require people to do this inline? Technically, it can be done in one "not large" statement. E.g.

```
let formatted = IDLValue::try_from_candid_type(&value)
  .map(|value| value.to_string())
  // Optionally, .map_err(...)
  ?;
```

Admittedly, this is not memorable at all though.

To reiterate, I am not strongly against what you did; do as you see fit. To me, this case is borderline. Just my two cents.

Reply:
This took me an hour to find, so I think I want to have the helper despite the small implementation.